app/services/highlighter/index.ts

```diff
@@ -256,21 +282,21 @@ export class HighlighterService extends StatefulService<IHighligherState> {
         // Aero 15 test clips
         // path.join(CLIP_DIR, '2021-05-12 12-59-28.mp4'),
         path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-20.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-29.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-41.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-49.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-58.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-14-03.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-14-06.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-30-53.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-29.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-41.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-49.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-13-58.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-14-03.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-14-06.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-30-53.mp4'),
         path.join(CLIP_DIR, 'Replay 2021-03-30 14-32-34.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-34-33.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-34-48.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-03.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-23.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-51.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-36-18.mp4'),
-        path.join(CLIP_DIR, 'Replay 2021-03-30 14-36-30.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-34-33.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-34-48.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-03.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-23.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-35-51.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-36-18.mp4'),
+        // path.join(CLIP_DIR, 'Replay 2021-03-30 14-36-30.mp4'),
```

Comment:
Why did you comment again these files?

Reply:
These files exist on my PC only.  I use it for fast testing, and I comment and uncomment based on how many clips I need to test with.  I'll remove them eventually.  Sorry for the diff noise.